### ISSUE TITLE
Ansibullbot incorrectly marking PRs as having "stale_ci"

### ISSUE THREAD

**Issue by gundalow** [ansible-community/collection_bot#4]
The CI run was still in progress and the Ansibullbot marked the PR as stale_ci

See for example:

ansible-collections/amazon.aws#142 (comment)

@tremble @jillr Copy of https://github.com/ansible/ansibullbot/issues/1420

**Comment by gundalow** [comment]
I've seen this when I forgot to update the Shippable Project ID.

It looks like `stale_ci` has been added to all PRs, though some have been removed by hand.

**Comment by jillr** [comment]
Yeah I'm unsure what the correct shippable key is.  Last night I noticed the one that the playbooks generated on the cloud bot is different than what lives in the ansibullbot-com.eng.ansible.com bot.  

I also found a separate issue with the ansibullbot-cloud.eng.ansible.com bot which meant it wasn't properly running over all collections it ought to be so there was a gap yesterday where the bot wasn't actually triaging that repo.  I have a work around in place since around 2400UTC last night though.
